js/src/components/free-listings/configure-product-listings/shipping-time/shipping-time-setup/getCountriesTimeArray.js

```diff
@@ -7,15 +12,18 @@
  * const shippingTimes = [
  *     {
  *         countryCode: 'US',
- *         time: 10
+ *         time: 10,
+ *		   maxTime: 20,
  *     },
  *     {
  *         countryCode: 'AU',
- *         time: 10
+ *         time: 10,
+ *		   maxTime: 20,
  *     },
  *     {
  *         countryCode: 'CN',
- *         time: 15
+ *         time: 15,
+ *		   maxTime: 22,
```

Comment:
Avoid mixed indentation chars.

```suggestion
 *         maxTime: 22,
```